client/js/components/main/assessment.jsx

```diff
@@ -113,6 +118,45 @@ export class Assessment extends React.Component{
   }
 
   /**
+   * Determine if user should be allowed to go to next questions or not
+   *
+   * unlockNext - Global setting controlling when next button should
+   * be available to the user. Should be one of "ON_CORRECT", "ON_ANSWER_CHECK",
+   * "ALWAYS"
+   *
+   * currentItem - The index of the current item in the array of items
+   *
+   * questionsPerPage - Global setting controller how many items should be
+   * displayed at once.
+   *
+   * checkedResponses - Array of all graded responses
+   *
+   * return - True if user is allowed to move onto next set of
+   * questions. False otherwise.
+   */
+  getNextUnlocked(unlockNext, currentItem, questionsPerPage, checkedResponses){
```

Comment:
Any ideas for a better name for this method? It determines if a user should be allowed to move onto the next set of questions or not.